hnn_core/tests/test_gui.py

```diff
@@ -0,0 +1,10 @@
+# Authors: Huzi Cheng <hzcheng15@icloud.com>
+from hnn_core.gui import run_hnn_gui
+import os
```

Comment:
import order ... put the `os` import first and then the `hnn_core` import. Also a space between your name and the imports